pkg/obfuscator/ssh.go

```diff
@@ -0,0 +1,67 @@
+package obfuscator
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/openshift/must-gather-clean/pkg/schema"
+	"golang.org/x/crypto/ssh"
+)
+
+const (
+	// consistentSSHTemplate refers to a static replacement for any identified SSH value
+	consistentSSHTemplate          = "x-%010d-x"
+	maximumSupportedObfuscationSSH = 9999999999
+)
+
+var (
+	sshKeyPattern = regexp.MustCompile(`(ssh-rsa AAAAB3NzaC1yc2|ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNT|ecdsa-sha2-nistp384 AAAAE2VjZHNhLXNoYTItbmlzdHAzODQAAAAIbmlzdHAzOD|ecdsa-sha2-nistp521 AAAAE2VjZHNhLXNoYTItbmlzdHA1MjEAAAAIbmlzdHA1Mj|ssh-ed25519 AAAAC3NzaC1lZDI1NTE5|ssh-dss AAAAB3NzaC1kc3)[0-9A-Za-z+/]+[=]{0,3}( .*)?$`)
+)
+
+type sshObfuscator struct {
+	ReplacementTracker
+	replacements []replacementGenerator
+}
+
+func (o *sshObfuscator) Path(s string) string {
```

Comment:
@DhritiShikhar I was under the impression that this should be implemented using CFE-81 (hence the issue dependency), can you let me know where you are with that?